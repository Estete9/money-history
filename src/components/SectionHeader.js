import PropTypes from 'prop-types';

function SectionHeader({ name, value }) {
  return (
    <section>
      <img src="/" alt="section header" />
      <div>
        <p>{name}</p>
        <p>{value}</p>
      </div>
    </section>
  );
}

SectionHeader.propTypes = {
  name: PropTypes.string.isRequired,
  value: PropTypes.string.isRequired,
};

export default SectionHeader;
