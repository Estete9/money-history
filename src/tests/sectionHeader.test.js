import renderer from 'react-test-renderer';
import React from 'react';
import SectionHeader from '../components/SectionHeader';

test('section header renders correctly', () => {
  const mockValues = {
    countryName: 'United States Dollar',
    value: 1,
    symbol: 'usd',
  };
  const component = renderer.create(
    <SectionHeader
      symbol={mockValues.symbol}
      value={mockValues.value}
      countryName={mockValues.countryName}
    />,
  );
  const sectionHeaderTree = component.toJSON();

  expect(sectionHeaderTree).toMatchSnapshot();
});
