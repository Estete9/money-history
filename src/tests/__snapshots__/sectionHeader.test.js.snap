// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`section header renders correctly 1`] = `
<section
  className="sectionHeader"
>
  <div
    className="sectionHeaderImgWrapper"
  >
    <img
      alt="currency icon"
      className="sectionHeaderImg"
      src="svg-mock"
    />
  </div>
  <div
    className="sectionHeaderContentWrapper"
  >
    <h1>
      $ 1.0558
    </h1>
    <p>
      United States Dollar
    </p>
    <p>
      â‚¬1 euro
    </p>
  </div>
</section>
`;
